spec/latest/json-ld/index.html

```diff
@@ -2317,7 +2317,7 @@ <h2>Scoped Contexts</h2>
     value">values</a> of properties defined using that <a>term</a>. This allows
     values to use <a>term definitions</a>, <a>base IRI</a>,
     <a>vocabulary mapping</a> or <a>default language</a> which is different from the
-    <a>node object</a> they are contained in, in exactly the same was as if the
+    <a>node object</a> they are contained in, in as if the
```

Comment:
Just to be picky... shouldn't it be "<a>context</a> was specified"? Or do we always use 'context' in plural?